runtime/doc/usr_04.txt

```diff
@@ -201,7 +201,8 @@ change.  Then you can use Visual mode.
 You start Visual mode by pressing "v".  You move the cursor over the text you
 want to work on.  While you do this, the text is highlighted.  Finally type
 the operator command.
-   For example, to delete from halfway one word to halfway another word:
+   For example, to delete from halfway through one word to halfway through
+   another word:
```

Comment:
Changing:
`to delete from halfway through one word to halfway through another word:`
->
`to delete from the middle of a word to the middle of another:` (fits on one line)